src/mraa.c

```diff
@@ -126,6 +158,7 @@ imraa_init()
 #error mraa_ARCH NOTHING
 #endif
 
+jsonloaded:
```

Comment:
let's replace this by a if (platform_type == MRAA_NULL_PLATFORM) and set platform_type by default to NULL